### DISCUSSION TITLE
Delete key to blank current cell

### DISCUSSION THREAD

**Discussion by DKMiller71** [viresh-ratnakar/exolve#94] [Ideas]
I think this would be a good option to blank the current cell without navigation, especially with rebus cells:

Right after the backspace `(key == 8)` section:

```
  if (key == 46) { // delete key
    let gridCell = this.currCell()
    if (gridCell && !gridCell.prefill) { 
      this.clearCell(this.currRow, this.currCol)
      this.updateAndSaveState()
    }
    return true
  }
```

**Comment by viresh-ratnakar** [comment]
This sounds good to me. Perhaps you can just initiate a Pull Request? If you don't have the time, then I can make the changes, no worries. I just think it would be good to leave a credit-trail.

The following additional would be needed:

- Make the condition be `if (gridCell && gridCell.isLight && !gridCell.prefill) {`
- Updating the `README.md` file to describe the feature briefly.
- Updating the `tools-msg` in-puzzle documentation [somewhere here](https://github.com/viresh-ratnakar/exolve/blob/d22b3d29fb4bf726dd3d465be6af2902d2b47c75/exolve-m.js#L353)
- Updating the stand-alone file `exolve.html`
- Adding a minor version change description to `CHANGELOG.md`

**Reply by DKMiller71** [reply]
So... first time I've tried updating a github repository that's not mine... hope I did it right.
